packages/flutter/test/cupertino/dialog_test.dart

```diff
@@ -76,26 +76,33 @@ void main() {
   });
 
   testWidgets('Dialog default action styles', (WidgetTester tester) async {
-    await tester.pumpWidget(const CupertinoDialogAction(
+    await tester.pumpWidget(wrap(const CupertinoDialogAction(
       isDefaultAction: true,
       child: const Text('Ok'),
-    ));
+    )));
 
     final DefaultTextStyle widget = tester.widget(find.byType(DefaultTextStyle));
 
     expect(widget.style.fontWeight, equals(FontWeight.w600));
   });
 
   testWidgets('Default and destructive style', (WidgetTester tester) async {
-    await tester.pumpWidget(const CupertinoDialogAction(
+    await tester.pumpWidget(wrap(const CupertinoDialogAction(
       isDefaultAction: true,
       isDestructiveAction: true,
       child: const Text('Ok'),
-    ));
+    )));
 
     final DefaultTextStyle widget = tester.widget(find.byType(DefaultTextStyle));
 
     expect(widget.style.fontWeight, equals(FontWeight.w600));
     expect(widget.style.color.red, greaterThan(widget.style.color.blue));
   });
 }
+
+Widget wrap(Widget child) {
```

Comment:
I liked the `boilerplate` name you used elsewhere.

Reply:
Changed wrap to boilerplate.